openwisp_network_topology/tests/test_api.py

```diff
@@ -85,6 +85,23 @@ def test_receive(self):
         self.assertEqual(self.node_model.objects.count(), 2)
         self.assertEqual(self.link_model.objects.count(), 1)
 
+    def test_receive_with_deprecated_url(self):
+        self._set_receive()
+        self.node_model.objects.all().delete()
+        data = self._load('static/netjson-1-link.json')
+        t = self.topology_model.objects.first()
+        path = f'/api/v1/receive/{t.pk}/?key=test'
```

Comment:
please use `reverse()`, do not hardcode URLs

Reply:
@nemesisdesign since we have two urls mapping to same view (because we are supporting backward compatibility), `reverse` will get confused which url to use since both of them have the same view name.

Reply:
@NoumbissiValere the solution is to change the old URL to have `receive_topology_deprecated` and use `reverse` here.